docs/src/configuration/app/hooks/vary-hooks-by-environment.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Vary hooks by environment
```

Comment:
The name of the page might be difficult to understand for foreigners. Maybe we could try to come up with something easier?

Reply:
Yes. I wasn't super happy with it before. Will think on it and suggest something different.